full-stack-tests/native-app/src/frontend/NativeAppTest.ts

```diff
@@ -31,9 +31,9 @@ export class NativeAppTest {
       this.imodelCloudEnv = new IModelHubCloudEnv();
     }
 
-    const project = await this.imodelCloudEnv.contextMgr.queryProjectByName(await AuthorizedFrontendRequestContext.create(), props.projectName);
-    assert(project && project.wsgId);
-    return project.wsgId;
+    const project = await this.imodelCloudEnv.contextMgr.getITwinByName(await AuthorizedFrontendRequestContext.create(), props.projectName);

```

Comment:
nit: wonder if this and the function and really any place that uses "Project" should be replaced with "ITwin"